webextensions/api/webRequest.json

```
@@ -398,6 +398,25 @@
               }
             }
           },
+          "json": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
```

As per https://github.com/mdn/browser-compat-data/pull/26512#issuecomment-2833897874:

```suggestion
                  "version_added": "138"
```